AssetTools.NET/Extra/AssetsReplacer/Texture2DAssetReplacer.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace AssetsTools.NET.Extra
+{
+    public abstract class Texture2DAssetReplacer : SerializingAssetReplacer
```

Comment:
The intention is indeed that the library user inherits this class and provides the RGBA data when the asset is about to be serialized (for example, by reading a PNG file and converting it to a Bitmap).

It would of course be more user friendly to offer such file loading functionality in the class itself so that it wouldn't have to be abstract. However, as it's part of a .NET Standard library, we can't use System.Drawing.Image.